packages/compass-e2e-tests/tests/collection-explain-plan-tab.test.ts

```diff
@@ -45,23 +45,24 @@ describe('Collection explain plan tab', function () {
   });
 
   it('shows a loading state while explain is running', async function () {
-    // Popuplate existing collection with some more data
-    await createNumbersCollection(collectionName, 10_00_000);
-    await browser.expandOptions(tabName);
-    await browser.setSort(tabName, '{ i: -1 }');
+    await browser.runFindOperation(
+      'Explain Plan',
+      '{ $where: "function() { sleep(10000); return true; }" }',
```

Comment:
```suggestion
      '{ $where: function() { sleep(10000); return true; }  }',
```
Shouldn't this be an actual JS function instead of string? Or is it same?

Reply:
As [documentation](https://www.mongodb.com/docs/manual/reference/operator/query/where/) mentions, it can be either: 

> `{ $where: <string|JavaScript Code> }`

In our case it makes even less difference because mongodb-query-parser will convert functions to string when parsing:

![image](https://user-images.githubusercontent.com/5036933/225244229-9cce77e2-f278-40f0-9e72-17c7af4d45d7.png)